src/evict/evict_lru.c

```diff
@@ -317,112 +319,33 @@ __evict_server(WT_SESSION_IMPL *session, bool *did_work)
 }
 
 /*
- * __evict_workers_resize --
- *	Resize the array of eviction workers (as needed after a reconfigure).
- *	We don't do this during the reconfigure because the eviction server
- *	thread owns these structures.
- */
-static int
-__evict_workers_resize(WT_SESSION_IMPL *session)
-{
-	WT_CONNECTION_IMPL *conn;
-	WT_DECL_RET;
-	WT_EVICT_WORKER *workers;
-	size_t alloc;
-	uint32_t i, session_flags;
-
-	conn = S2C(session);
-	workers = NULL;			/* -Wconditional-uninitialized */
-
-	if (conn->evict_workers_alloc < conn->evict_workers_max) {
-		alloc = conn->evict_workers_alloc * sizeof(*workers);
-		WT_RET(__wt_realloc(session, &alloc,
-		    conn->evict_workers_max * sizeof(*workers),
-		    &conn->evict_workctx));
-		workers = conn->evict_workctx;
-	}
-
-	for (i = conn->evict_workers_alloc; i < conn->evict_workers_max; i++) {
-		/*
-		 * Eviction worker threads get their own session.
-		 * Eviction worker threads may be called upon to perform slow
-		 * operations for the block manager.
-		 *
-		 * Eviction worker threads get their own lookaside table cursor
-		 * if the lookaside table is open.  Note that eviction is also
-		 * started during recovery, before the lookaside table is
-		 * created.
-		 */
-		session_flags = WT_SESSION_CAN_WAIT;
-		if (F_ISSET(conn, WT_CONN_LAS_OPEN))
-			FLD_SET(session_flags, WT_SESSION_LOOKASIDE_CURSOR);
-		WT_ERR(__wt_open_internal_session(conn, "eviction-worker",
-		    false, session_flags, &workers[i].session));
-		workers[i].id = i;
-
-		if (i < conn->evict_workers_min) {
-			++conn->evict_workers;
-			F_SET(&workers[i], WT_EVICT_WORKER_RUN);
-			WT_ERR(__wt_thread_create(workers[i].session,
-			    &workers[i].tid, __evict_thread_run,
-			    workers[i].session));
-		}
-	}
-
-err:	conn->evict_workers_alloc = conn->evict_workers_max;
-	return (ret);
-}
-
-/*
  * __wt_evict_create --
  *	Start the eviction server thread.
  */
 int
 __wt_evict_create(WT_SESSION_IMPL *session)
 {
 	WT_CONNECTION_IMPL *conn;
-	uint32_t session_flags;
+	WT_DECL_RET;
```

Comment:
Where are we using `ret` in here?  WT_RET doesn't need it.  So I think we can get rid of that and return (0) at the end.